resources/assets/js/models/types.ts

```diff
@@ -50,6 +50,15 @@ export interface Job {
   fr: JobTranslation;
 }
 
+export enum JobStatus {
```

Comment:
Just following our previous patterns, it would be more appropriate to put this in resources\assets\js\models\lookupConstants.ts

Reply:
My bad, I'll keep that in mind next time.